modules/ROOT/pages/salesforce/salesforce-connector-use-cases.adoc

```diff
@@ -0,0 +1,359 @@
+= Salesforce - Use Cases
```

Comment:
Customers don't search on "use cases". They search with some vigor for examples. So let's call these examples.